inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -41,29 +46,78 @@ std::string ConvolutionLayerTest::getTestCaseName(const testing::TestParamInfo<c
 
 void ConvolutionLayerTest::SetUp() {
     convSpecificParams convParams;
-    std::vector<size_t> inputShape;
-    auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) =
+    std::vector<std::pair<size_t, size_t>> inputShape;
+    std::vector<InferenceEngine::SizeVector> targetShapes;
+    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetShapes, targetDevice) =
         this->GetParam();
-    ngraph::op::PadType padType;
-    InferenceEngine::SizeVector kernel, stride, dilation;
-    std::vector<ptrdiff_t> padBegin, padEnd;
-    size_t convOutChannels;
+    for (auto&& targetShape : targetShapes) {
+        targetStaticShapes.emplace_back(targetShape);
+    }
+    inputDynamicShape = FuncTestUtils::PartialShapeUtils::vec2partialshape(inputShape, targetStaticShapes[0]);
     std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
+
+    setTargetStaticShape(targetStaticShapes[0]);
+    function = makeConvolution("convolution");
+    functionRefs = makeConvolution("convolutionRefs");
```

Comment:
`functionRefs = makeConvolution("convolutionRefs");` --> `functionRefs = ngraph::clone_function(function); functionRefs->set_friendly_name("convolutionRefs");`

In this case we can put ngraph function creation from `ConvolutionLayerTest::makeConvolution` to the `SetUp()` function

Reply:
Codes are updated as you commented.